platform-emf/src/main/java/com/softicar/platform/emf/attribute/field/enums/table/row/EmfEnumTableRows.java

```diff
@@ -0,0 +1,25 @@
+package com.softicar.platform.emf.attribute.field.enums.table.row;
+
+import com.softicar.platform.common.core.entity.IEntity;
+import com.softicar.platform.common.core.i18n.IDisplayString;
+import com.softicar.platform.common.core.i18n.IDisplayable;
+import com.softicar.platform.db.runtime.enums.IDbEnumTableRow;
+
+/**
+ * Utility methods for {@link IDbEnumTableRow} objects.
+ *
+ * @author Oliver Richers
+ */
+public class EmfEnumTableRows {
+
+	public static IDisplayString toDisplay(IDbEnumTableRow<?, ?> row) {
```

Comment:
We could use some Javadoc here (parameter non-null, etc)

Reply:
added